lambdas/tests/unit/handlers/contract/routes/access-token-router.ts

```diff
@@ -26,23 +30,68 @@ class AccessTokenRouter {
         this.router.post("/", this.handlePostRequest.bind(this));
     }
 
-    private async handlePostRequest(req: Request, res: Response, next: NextFunction) {
+    private async handlePostRequest(_req: Request, res: Response, next: NextFunction) {
         try {
-            const event = res.locals.event;
+            const body = res.locals.event.body;
+            const bodyAsJson = res.locals.json;
+            const componentId = res.locals.componentId;
 
-            const clientConfig = new Map<string, string>();
-            clientConfig.set("code", this.code);
-            clientConfig.set("redirectUri", this.redirectUri);
+            const lambda = CreateAccessTokenLambda(
+                new MockSSMProvider({
```

Comment:
Would we expect different AccessTokenLambdas to have different ssm params? I'm interestied in why we do this here rather than in the `CreateAccessTokenLambda` function

Reply:
It was originally, in `CreateAccessTokenLambda` but the `componentId` is different depending on the CRI, so we refactor this

Reply:
Nice like that change 👍